bin/apisix

```diff
@@ -799,6 +799,8 @@ version:    print the version of apisix
 ]])
 end
 
+
+local is_checked_admin_key
```

Comment:
`is_checked_admin_key` is confusing, please take a better name.

Reply:
`checked_admin_key`, any better name suggestions?